### DISCUSSION TITLE
creating executable files for both windows and android

### DISCUSSION THREAD

**Discussion by Winstonjames2** [GreaterWMS/GreaterWMS#362] [General]
I am trying to create executable files for both windows and android. I tried to do with pyinstaller and I have done several research but I got various errors, and I am seeking help of creating exe files. Please Can you tell me how I can create them.

**Comment by Winstonjames2** [comment]
please someone lighten me I have been searching for it and I haven't found the answer

**Comment by Winstonjames2** [comment]
I totally get it now everything. anyway waiting for GWMS ver3. Thanks!

**Reply by Singosgu** [reply]
GWMS v3 will fully support new framework Bomiot

why you want to use pyinstaller？

**Reply by Winstonjames2** [reply]
right now i understand how the project works and I don't want pyinstaller anymore. Thanks for your effort.
